src/main/java/seedu/address/logic/Messages.java

```diff
@@ -45,6 +45,9 @@ public static String format(Person person) {
                 .append(person.getAddress())
                 .append("; Tags: ");
         person.getTags().forEach(builder::append);
+        builder.append("; Upcoming: ")
+                .append(person.getUpcoming())
+                .append(";");
```

Comment:
@tanyunchao IG no need to append ";" at the end since it will be handled by the next field as seen in the next line.

Reply:
ooh good point, initially didnt have the field after therefor needed to add it